docs/additional-examples/evm-swap.md

```diff
@@ -0,0 +1,656 @@
+# Cross Chain EVM Swap
+In this section, we will demonstrate how you can use PKPs and Lit Actions to perform a cross-chain swap. There are three parts to this flow:
+
+- Understanding Lit PKPs as escrow
+- Lit Action
+- Implementing Mint/Grant/Burn with SDK
+
+## 1) Understanding Lit PKPs as escrow
+
+A Private key is meant for signing transactions which are executed on the blockchain, a network verifies if the transaction is a legitimate one and then executes it. With Lit Network we can generate a special type of private keys which can be programmed to sign transactions only when certain conditions are met. With this, we can create several escrow mechanisms, and we will look into how we can execute a cross-chain swap.
+
+### A swap example would be:
+Suppose Alice holds 100 token A on Base and Bob holds 100 token B on Chronicles and they both agree to a swap of,
+- 4 token A from Alice to Bob on Base
+- 8 token B from Bob to Alice on Chronicles
+
+We have an [example code](https://github.com/anshss/lit-evm-swap) for the below solution.
+
+### How does this work?
+When you generate a PKP on the LIT Network, a PKP NFT is first minted to your wallet, representing your ownership over the underlying private key residing on the LIT Network. This PKP NFT is responsible for managing auth methods over the private key.
+
+Lit Actions are programmable code written in Javascript which can run on the LIT Network. We can program a Lit Action so that if certain permissions are met for a pkp, it must sign specific transactions that can be sent to the blockchain. We need to grant a Lit Action as a permitted action to the PKP to allow the Lit Action to generate signatures with it.
+
```

Comment:
> When you generate a PKP on the LIT Network, a PKP NFT is first minted to your wallet, representing your ownership over the underlying private key residing on the LIT Network. This PKP NFT is responsible for managing auth methods over the private key.

Lit Actions are programmable code written in Javascript which can run on the LIT Network. We can program a Lit Action so that if certain permissions are met for a pkp, it must sign specific transactions that can be sent to the blockchain. We need to grant a Lit Action as a permitted action to the PKP to allow the Lit Action to generate signatures with it.

These are all details we can abstract away from this guide imo. We can link to the intro pages for pkps and actions if folks are interested in learning about how they work generally. For the purpose of this guide, i think we should focus on the specific use of these in the implementation itself. So starting with "we first mint a PKP and then assign a Lit Action to it using the m/g/b method [link to code snippet]. Next, we ..." Going through the steps sequentially

Reply:
I've replaced this section with 
https://github.com/LIT-Protocol/docs/pull/330/commits/30185e8fdd79532766306cf2736f33c5726e9f50#diff-0e36021eb2e9b13aa34e50ec77474d75d50bab8d18cba35c954841b81d9efee7R19